UnitTests/GitCommands.Tests/Git/Commands/GitCommandHelpersTest.cs

```diff
@@ -167,28 +167,30 @@ public void TestUnsetStagedStatus()
         [Test]
         public void SubmoduleSyncCmd()
         {
-            Assert.AreEqual("submodule sync \"foo\"", GitCommandHelpers.SubmoduleSyncCmd("foo").Arguments);
-            Assert.AreEqual("submodule sync", GitCommandHelpers.SubmoduleSyncCmd("").Arguments);
-            Assert.AreEqual("submodule sync", GitCommandHelpers.SubmoduleSyncCmd(null).Arguments);
+            string config = "-c protocol.file.allow=always";
+            Assert.AreEqual($"{config} submodule sync \"foo\"", GitCommandHelpers.SubmoduleSyncCmd("foo").Arguments);
+            Assert.AreEqual($"{config} submodule sync", GitCommandHelpers.SubmoduleSyncCmd("").Arguments);
+            Assert.AreEqual($"{config} submodule sync", GitCommandHelpers.SubmoduleSyncCmd(null).Arguments);
         }
 
         [Test]
         public void AddSubmoduleCmd()
         {
+            string config = "-c protocol.file.allow=always";
```

Comment:
config in settings is related to submodule. Same in tests?

Reply:
The point is so if another con fig is needed, you just add to that config and you don't have to repeat the change per Assert.

Reply:
I am working on making tests green now .  Later I'll move to test only config but I wanted to confirm app  behaved and tests succeeded.  Then looking at always vs user and such mentioned in issue.